asset/js/setup.js

```diff
@@ -95,6 +95,21 @@ function setupSiteNav() {
   );
 }
 
+function setupReadingProgress() {
+  const h = document.documentElement;
```

Comment:
Could we use more descriptive variable names here?

Reply:
I've removed them altogether. They're not really required.